pydy/tests/test_utils.py

```diff
@@ -18,3 +22,61 @@ def test_sympy_equal_to_or_newer_than():
     if parse_version(SETUPTOOLS_VERSION) >= parse_version('8.0'):
         with assert_raises(ValueError):
             sympy_equal_to_or_newer_than('0.7.7', '0.7.6-git')
+
+def test_mass():
+
+    mass = Mass()
+    assert mass.symbol == Symbol('m')
+    assert mass.value == 1
+
+    body = Body('body')
+    body_mass = Mass(body.name)
+    assert body_mass.symbol == Symbol(body.name + "_mass")
+
+    body_mass_value = Mass(body.name, 4)
+    assert body_mass_value.value == 4
+
+def test_spring_constant():
+
+    spring_constant = SpringConstant()
+    assert spring_constant.constant == Symbol('k')
+    assert spring_constant.value == 1
+
+    body = Body('body')
+    body_mass = Mass(body.name)
+    assert body_mass.symbol == Symbol(body.name + "_mass")
+
+    body_mass_value = Mass(body.name, 4)
+    assert body_mass_value.value == 4
+
+def test_length():
+
+    length = Length()
+    assert length.constant == Symbol('l')
+    assert length.value == 1
+
+    body = Body('body')
+    body_length = Length(body.name)
+    assert body_length.symbol == Symbol(body.name + "_length")
+
+    body_length_value = Length(body.name, 2)
+    assert body_length_value.value == 2
+
+def test_gravitational_constant():
+
+    gravitational_constant = GravitationalConstant()
```

Comment:
All of the test_utils is redundant for now. I am trying to push atomic PRs. All of this will be removed there. Thanks